keras_hub/src/models/efficientnet/convbnact_test.py

```diff
@@ -0,0 +1,22 @@
+import keras
```

Comment:
rename file to be more readable

Reply:
this conflicts w/ the below review, so I chose the later review.